content/docs/next-release-v2/deployment.md

```diff
@@ -37,109 +36,55 @@ There are orchestration templates for running Jaeger with:
-  -p 9411:9411 \
-  jaegertracing/all-in-one:{{< currentVersion >}}
-```
-
-You can navigate to `http://localhost:16686` to access the Jaeger UI.
-
-## Agent
-
-{{< warning >}}
-**jaeger-agent** is [deprecated](https://github.com/jaegertracing/jaeger/issues/4739). The OpenTelemetry data can be sent from the OpenTelemetry SDKs (equipped with OTLP exporters) directly to **jaeger-collector**. See the [Architecture](../architecture) page for alternative deployment options.
-{{< /warning >}}
+2777  | HTTP     | expvar port for process level metrics per the Go standards
+8888  | HTTP     | metrics port for exposing metrics which can be scraped with Prometheus compatible systems at `/metrics`
+8889  | HTTP     | ingester port for reading data from Kafka topics and writing to a supported backend
+13133 | HTTP     | Healthcheck port via the `healthcheckv2` extension
 
-**jaeger-agent** is designed to receive tracing data in Thrift format over UDP and run locally on each host, either as a host agent / daemon or as an application sidecar. **jaeger-agent** exposes the following ports:
+**jaeger** is stateless and thus many instances of **jaeger** can be run in parallel. **jaeger** instances require almost no configuration, except for storage location, such as:
 
-Port  | Protocol | Function
------ | -------  | ---
-6831  | UDP      | Accepts [jaeger.thrift][jaeger-thrift] in `compact` Thrift protocol used by most current Jaeger clients.
-6832  | UDP      | Accepts [jaeger.thrift][jaeger-thrift] in `binary` Thrift protocol used by Node.js Jaeger client (because [thriftrw][thriftrw] npm package does not support `compact` protocol).
-5778  | HTTP     | Serves SDK configs, namely sampling strategies at `/sampling` (see [Remote Sampling](../sampling/#remote-sampling)).
-5775  | UDP      | Accepts [zipkin.thrift][zipkin-thrift] in `compact` Thrift protocol (deprecated; only used by very old Jaeger clients, circa 2016).
-14271 | HTTP     | Admin port: health check at `/` and metrics at `/metrics`.
-
-It can be executed directly on the host or via Docker, as follows:
-
-```sh
-## make sure to expose only the ports you use in your deployment scenario!
-docker run \
-  --rm \
-  -p6831:6831/udp \
-  -p6832:6832/udp \
-  -p5778:5778/tcp \
-  -p5775:5775/udp \
-  jaegertracing/jaeger-agent:{{< currentVersion >}}
+Cassandra:
 ```
-
-### Discovery System Integration
-
-**jaeger-agent**s can connect point-to-point to a single **jaeger-collector** address, which could be
-load balanced by another infrastructure component (e.g. DNS) across multiple **jaeger-collector**s.
-**jaeger-agent** can also be configured with a static list of **jaeger-collector** addresses.
-
-On Docker, a command like the following can be used:
-
-```sh
-docker run \
-  --rm \
-  -p5775:5775/udp \
-  -p6831:6831/udp \
-  -p6832:6832/udp \
-  -p5778:5778/tcp \
-  jaegertracing/jaeger-agent:{{< currentVersion >}} \
-  --reporter.grpc.host-port=jaeger-collector.jaeger-infra.svc:14250
+  jaeger_storage:
```

Comment:
We have dedicated pages for each storage backend now, I would rather consolidate everything about that backend there, including config examples. This page could be just about different deployment roles, how they are different, the ports, the APIs exposed.

Reply:
I put the snippets here just to provide some guidance. If you want me to just point to the storage pages, I can do that. Please clarify.

Reply:
Yes, I would put them in the storage pages and link from here. I don't think we should focus so much on the backends in this top page.